pax-core/src/engine.rs

```diff
@@ -643,69 +263,59 @@ impl PaxEngine {
     /// 3. Render:
     ///     a. find lowest node (last child of last node)
     ///     b. start rendering, from lowest node on-up, throughout tree
-    pub fn tick(&self, rcs: &mut HashMap<String, Box<dyn RenderContext>>) -> Vec<NativeMessage> {
-        let root_component_instance = Rc::clone(&self.main_component);
-        let mut z_index = LayerId::new(None);
+    pub fn tick(&mut self) -> Vec<NativeMessage> {
```

Comment:
Please propagate this refactor to `pax-chassis-common` and get it building again

Reply:
Got it building! Isn't rendering text, is this expected?

Reply:
Previously it was rendering text (see screenshot from mid-Dec) but not a critical issue to fix rn.  Probably related to changes in native message ordering, or possibly occlusion (which is yet-unimplemented on macos/ios).  We can fix this later.

<img width="3839" alt="Pasted Graphic 1" src="https://github.com/paxproject/pax/assets/2100885/2916a1f5-822e-46ac-876d-296d6bdbf767">